### PR TITLE
Portfolio: finish mobile version 

### PR DESCRIPTION
**Pull request by hafedEfheij**
In this Portfolio: finish the mobile version of this model I have.
-Update the last version of Portfolio mobile first.
-added about section and design as requested in the project requirement.
-added headlines and write paragraphs and designed them with CSS

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -69,38 +129,198 @@ header {
...
+  background: #fff;
+  border-radius: 8px;
+  width: max-content;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  padding: 12px;
+}
```
Comment (MaganaAsiati) [inline styles.css]: - Kindly follow the button styling: Active, Hover, and Pressed as shown in the figma design :+1: .

* Apart from being semantically correct, the `a` tag will let you connect to other sections of the website, using href.1: 

image

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 

Hi @hafedEfheij 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlonzoLeroneThisCloseGIF

## Highlights
✅ No linter error
✅ README is professional

## Required Changes ♻️


- Kindly make sure the cards are centered in the middle of the page. The margin left and right are the same which will make your design stick more to the figma design :+1: 

## Your design 

image


## Figma design 

image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: styles.css (hunk 1 above)
Comment (hafedEfheij) [reply styles.css]: @MaganaAsiati Hello thanks for your review, but I did not understand your second review about the button I think I  did the same colors as Figma, can you explain more in detail?

**Review by franclobo — APPROVED** [review]
Hi @hafedEfheij, 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Github flow ✔️ 
- Professional README file ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**